Travis

WahlbergRu opened the pull request:
test travis

Mariya-Z commented:
I built the extension in this branch and added it to Chrome extensions. JDN didn't appear in devTools. 
I've changed files in dist folder in order to fix it:
1. Moved script from index.html into a separate file
2. Added this script into index.html and manifest.json
3. Copied folder with icons
Hope it'll help with fixing build settings

WahlbergRu commented:
> I built the extension in this branch and added it to Chrome extensions. JDN didn't appear in devTools.
> I've changed files in dist folder in order to fix it:
> 
> 1. Moved script from index.html into a separate file
> 2. Added this script into index.html and manifest.json
> 3. Copied folder with icons
>    Hope it'll help with fixing build settings

FIxed
